net/netatalk/Makefile

```diff
@@ -43,34 +40,35 @@ define Package/netatalk/decription
   Protocol Suite.
 endef
 
-TARGET_CFLAGS += -std=gnu99
-
-CONFIGURE_ARGS += \
-	--disable-afs \
-	--disable-hfs \
-	--enable-debugging \
-	--disable-shell-check \
-	--disable-timelord \
-	--disable-a2boot \
-	--disable-cups \
-	--disable-tcp-wrappers \
-	--with-cnid-default-backend=dbd \
-	--with-bdb="$(STAGING_DIR)/usr/" \
-	--with-libevent=no \
-	--with-libgcrypt-dir="$(STAGING_DIR)/usr" \
-	--with-ssl-dir="$(STAGING_DIR)/usr" \
-	--with-uams-path="/usr/lib/uams" \
-	--without-acls \
-	--without-kerberos \
-	--without-mysql \
-	--with-mysql-config=false \
-	--without-pam \
-	--disable-admin-group \
-	--disable-srvloc \
-	--disable-zeroconf \
-	$(if $(CONFIG_SHADOW_PASSWORDS),--with-shadow,--without-shadow) \
-	--without-dtrace \
-	--without-ldap
+MESON_ARGS += \
+	-Dwith-appletalk=false \
+	-Dwith-zeroconf=false \
+	-Dwith-cups=false \
+	-Dwith-quota=false \
+	-Dwith-afpstats=true \
+	-Dwith-acls=false \
+	-Dwith-ldap=false \
+	-Dwith-cnid-default-backend=dbd \
+	-Dwith-cnid-dbd-backend=true \
+	-Dwith-cnid-mysql-backend=false \
+	-Dwith-pam=false \
+	-Dwith-tests=false \
+	-Dwith-testsuite=false \
+	-Dwith-bdb-path="$(STAGING_DIR)/usr/" \
+	-Dwith-unicode-data-path="$(STAGING_DIR)/../host/share/gnulib/tests/uniname/" \
```

Comment:
I think, we can do this even better. Not tested, so far.

	-Dwith-unicode-data-path="$(STAGING_DIR_HOST)/share/gnulib/tests/uniname/" \

Reply:
Addressed in third commit that enables AppleTalk features.